Userland/Libraries/LibWeb/XHR/XMLHttpRequest.idl

```diff
@@ -33,7 +36,7 @@ interface XMLHttpRequest : XMLHttpRequestEventTarget {
     undefined open(DOMString method, DOMString url);
     undefined open(ByteString method, USVString url, boolean async, optional USVString? username = {}, optional USVString? password = {});
     undefined setRequestHeader(DOMString name, DOMString value);
-    undefined send(optional XMLHttpRequestBodyInit? body = null);
+    undefined send(optional DocumentOrXMLHttpRequestBodyInit? body = null);
```

Comment:
We can write this according to the spec once https://github.com/SerenityOS/serenity/pull/16009 is merged